.gitignore

```diff
@@ -4,3 +4,9 @@ composer.phar
 
 ### Phive
 /tools/
+
```

Comment:
This changes are not needed. It's better to have such lines in the [global .gitignore](https://thoughtbot.com/blog/global-gitignore).